### DISCUSSION TITLE
Ubuntu

### DISCUSSION THREAD

**Discussion by almostheaven1863** [cryptosharks131/lndg#201] [General]
I am running an Ubuntu node and am having trouble installing LNDg. I already have docker installed. I think the issue is with editing the yaml file. 
I am not completely sure how to fill the volumes section out correctly. 
volumes:
      - /home/<user>/.lnd:/root/.lnd:ro
      - /home/<user>/<path-to>/lndg/data:/lndg/data:rw
   I changed the user name but am not sure if I need to change where it says root. 
   Also, my path to lndg is just /home/<user>/lndg. Not sure if there’s something else I need to be adding where it says <path-to>.

**Comment by cryptosharks131** [comment]
Usually those lines would look something like this:

- /home/<your_user>/.lnd:/root/.lnd:ro
- /home/<your_user>/lndg/data:/lndg/data:rw

Left side of 1st colon: path on your system (update this for your system)
Right side of 1st colon: path inside container (don't change this)

If you are unsure of the path, you can type `pwd` inside the directory you are trying to map to get the full path of the directory.

**Reply by almostheaven1863** [reply]
Thank you for the info. I will try this once I get home tonight. I was confused about what the : meant. Thanks for the explanation.

**Reply by almostheaven1863** [reply]
Yes, this worked. Thank you!
